snapshotter/devmapper/config.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package devmapper
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+
+	"github.com/docker/go-units"
+	"github.com/hashicorp/go-multierror"
+	"github.com/pkg/errors"
+)
+
+const (
+	configPathEnvName = "FIRECRACKER_CONTAINERD_DEVMAPPER_SNAPSHOTTER_CONFIG_PATH"
```

Comment:
We've used `FIRECRACKER_CONTAINERD_XXX` before and we have 2 different snapshotters.
It's long because I want it to be clear to what component (agent, runtime, naive snapshotter, devmapper snapshotter) this is related to. If you have better ideas feel free to share )

Reply:
ping @xibz do you any ideas or can be marked as solved?

Reply:
Obviously it should be `F9R_C8D_DM_S_CFG`. :see_no_evil:  (ignore me, that is not a serious suggestion)

More seriously, though, is it conceivable that somebody might use this snapshotter with a non-Firecracker runtime?

Reply:
> Obviously it should be F9R_C8D_DM_S_CFG. 🙈 (ignore me, that is not a serious suggestion)

Elegant solution I was looking for all these times :)

> More seriously, though, is it conceivable that somebody might use this snapshotter with a non-Firecracker runtime?

Potentially any runtime with block devices support should work. Do you think it makes sense not to use word `Firecracker` inside `snapshotters` package and have it more generic "block device snapshotter"?